src/main/resources/fabric.mod.json

```diff
@@ -13,17 +13,18 @@
   },
```

Comment:
While also out of scope, this pr could be improved by adding an issues argument to the contact section, allowing the Issues button to be available in the Mod Menu as well